<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSerial</class>
 <widget class="QWidget" name="FormSerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,8" columnminimumwidth="0,0">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPortName">
       <property name="text">
        <string>Port Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelBaudRate">
       <property name="text">
        <string>Baud Rate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDataBit">
       <property name="text">
        <string>Data Bit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelCheckBit">
       <property name="text">
        <string>Check Bit</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelStopBit">
       <property name="text">
        <string>Stop Bit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cBoxPortName"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cBoxBaudRate"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cBoxDataBit"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cBoxCheckBit"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cBoxStopBit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="8,2">
     <item>
      <widget class="QGroupBox" name="groupBoxReceive">
       <property name="title">
        <string>Receive</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="txtRecv"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSend">
       <property name="title">
        <string>Send</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPlainTextEdit" name="txtSend"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSend">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnSerialSwitch">
     <property name="text">
      <string>Open/Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxHexDisplay">
       <property name="text">
        <string>Hex Display</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnRecvSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRecvClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
